test/controllers/beneficial.owner.individual.controller.spec.ts

```diff
@@ -144,6 +152,20 @@ describe("BENEFICIAL OWNER INDIVIDUAL controller", () => {
       expect(resp.text).toContain(SANCTIONS_HINT_TEXT_THEY);
       expect(resp.text).toContain(TRUSTS_NOC_HEADING);
     });
+
+    test(`renders the ${BENEFICIAL_OWNER_INDIVIDUAL_PAGE} page with correct back link url when feature flag is off`, async () => {
```

Comment:
i would say what feature flag is off in the test description, e.g. when Redis removal flag is off.
Same for the 'on' ones.

Reply:
Done.